javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+
+var todos = [];
+
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
You should check if there is anything stored in `previousTodosJSON` before reassigning the value of `todos`. If there isn't anything there, the value will be `null`, which means you would be destroying the array.